index.html

```diff
@@ -0,0 +1,215 @@
+            cupiditate sequi odit nam doloribus magnam magni eaque consectetur
+            unde adipisci veniam praesentium, rerum aspernatur sit quis laborum
+            ut! Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio,
+            similique tempora molestias suscipit modi impedit nemo voluptas.
+            Magnam corrupti maiores quo delectus. Ratione, deleniti? Eius et
+            quisquam exercitationem ad similique.
+          </p>
+          <img
+            class="article__img"
+            src="https://via.placeholder.com/725x375?text=Article+Image"
+            alt="Placeholder Image"
+            width="725"
+            height="375"
+          />
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Odio,
+            similique tempora molestias suscipit modi impedit nemo voluptas.
+            Magnam corrupti maiores quo delectus. Ratione, deleniti? Eius et
+            quisquam exercitationem ad similique.
+          </p>
+        </div>
+      </article>
+    </main>
+    <aside class="related">
+      <h3 class="related__title">Alte articole relevante</h3>
+      <section class="related__wrapper">
+        <article class="related__item item">
+          <a href="#" class="item__link">
+            <h4 class="item__title">Titlu articol sportiv</h4>
+          </a>
+        </article>
+        <article class="related__item item">
+          <a href="#" class="item__link">
+            <h4 class="item__title">
+              Alt titlu articol sportiv pe trei randuri
+            </h4>
+          </a>
+        </article>
+        <article class="related__item item">
+          <a href="#" class="item__link">
+            <h4 class="item__title">
+              Încă un titlu articol sportiv pe trei randuri sau chiar mai mult
+              de 3 randuri
+            </h4>
+          </a>
+        </article>
+      </section>
+      <a href="#" class="related__action">Vezi toate articolele</a>
+    </aside>
+    <footer class="footer">
+      <div class="footer__social social">
+        <ul class="social__wrapper">
+          <li class="social__item">
+            <img
+              src="https://via.placeholder.com/130x35?text=Facebook"
+              alt="Logo of Facebook"
+              width="130"
+              height="35"
+              class="social__image"
+            />
+          </li>
+          <li class="social__item">
+            <img
+              src="https://via.placeholder.com/105x35?text=Twitter"
+              alt="Logo of Twitter"
+              width="105"
+              height="35"
+              class="social__image"
+            />
+          </li>
+        </ul>
+      </div>
+      <div class="footer__nav nav-secondary">
+        <ul class="nav-secondary__wrapper">
+          <li class="nav-secondary__item">
+            <a href="#" class="nav-secondary__link"> Biciclete </a>

```

Comment:
```suggestion
            <a href="#" class="nav-secondary__link">Biciclete </a>

```

Daca ai nevoie de spatiu inainte foloseste margin / padding in css. Spatiul lasat inainte poate avea diferite valori in diferite browsere.